stack.go

```diff
@@ -0,0 +1,30 @@
+package slogx
+
+import (
+	"fmt"
+	"log/slog"
+	"runtime"
+	"strings"
+)
+
+const (
+	KeyStack = "stack"
+	trim     = 3
+)
+
+// Stack returns a stack trace formatted as panic output.
+// It excludes a call of slogx.Stack() itself.
+func Stack() slog.Attr {
+	buf := make([]byte, 2048)
+	runtime.Stack(buf, false)
+
+	s := strings.Split(string(buf), "\n")
```

Comment:
To me it looks like a good idea to optimize this, because it's already too slow and making it even more slow with Split/Join and allocating extra slice… :disappointed: 
Here is example how to do this fast: https://go.dev/play/p/jppXtMB8d5P